keys.go

```diff
@@ -548,6 +555,38 @@ func (c *Client) wrap(ctx context.Context, idOrAlias string, plainText []byte, a
 	return pt, ct, nil
 }
 
+// WrapWithKeyVersion function supports KeyVersion Details, PlainText and Cyphertext in response
+func (c *Client) WrapV2(ctx context.Context, idOrAlias string, plainText []byte, additionalAuthData *[]string) (*KeyActionResponse, error) {
+	keysActionReq := &KeysActionRequest{}
+	keyActionRes := &KeyActionResponse{}
+
+	if plainText != nil {
+		_, err := base64.StdEncoding.DecodeString(string(plainText))
+		if err != nil {
+			return keyActionRes, err
+		}
+		keysActionReq.PlainText = string(plainText)
+	}
+
+	if additionalAuthData != nil {
+		keysActionReq.AAD = *additionalAuthData
+	}
+
+	keysAction, err := c.doKeysAction(ctx, idOrAlias, "wrap", keysActionReq)
+	if err != nil {
+		return keyActionRes, err
+	}
+
+	keyActionRes = &KeyActionResponse{
+		PlainText:  keysAction.PlainText,
+		CipherText: keysAction.CipherText,
+		KeyVersion: &KeyVersion{
+			ID: keysAction.KeyVersion.ID,
```

Comment:
Pls add a check for `keysAction.KeyVersion != nil` as it is a pointer. If it would be nil for any reason, in current implementation it would throw nil pointer dereference.